### PR TITLE
feat(organization): Prevent update name of organization

### PR DESCRIPTION
**Pull request by javirln**
This patch removes the option of changing the name of an organization. This removes all associated files with it.

Refs #980

### CODE REVIEW

File: app/controlplane/pkg/biz/organization.go [hunk 1]
```diff
@@ -144,47 +144,6 @@ func (uc *OrganizationUseCase) doCreate(ctx context.Context, name string, opts .
 	return org, nil
 }
 
-func (uc *OrganizationUseCase) Update(ctx context.Context, userID, orgID string, name *string) (*Organization, error) {
```
Comment (migmartri) [inline app/controlplane/pkg/biz/organization.go:147]: you probably need to also update the `ent` schema in `data` to make it `inmmutable`

**Review by migmartri — APPROVED** [review]
please sync with @tknura about the changes required in the UI :)

and don't forget to make also the changes in the `data` layer as stated inline in my comments. 

Thanks!

File: app/controlplane/pkg/biz/organization.go (hunk 1 above)
Comment (javirln) [reply app/controlplane/pkg/biz/organization.go:147]: Absolutely, I had the files no added to the stash...Submitting them, thanks!

**Comment by javirln** [comment]
> please sync with @tknura about the changes required in the UI :)
> 
> and don't forget to make also the changes in the `data` layer as stated inline in my comments.
> 
> Thanks!

Sure thing, I'll ping him tomorrow, I also have the PR on the UI side ready :D

Thanks :D

**Review by migmartri — APPROVED** [review]
Thanks

**Comment by jiparis** [comment]
Thanks. For another change, we might want to add a "Display name" or "Description" field that can be edited.

**Review by jiparis — APPROVED** [review]

**Review by migmartri — APPROVED** [review]

**Review by migmartri — APPROVED** [review]

**Pull request merged**